src/CAREUI/interactive/FiltersSlideover.tsx

```diff
@@ -35,18 +33,13 @@ export default function FiltersSlideover({
         <div className="flex items-center justify-between">
           <span className="text-lg font-bold">{t("filters")}</span>
           <div className="mr-2 flex items-center justify-end gap-1">
-            <ButtonV2
-              variant="danger"
-              ghost
-              onClick={onClear}
-              id="clear-filter"
-            >
+            <Button variant="warning" onClick={onClear} id="clear-filter">
               <CareIcon icon="l-filter-slash" className="text-lg" />
               <span>{t("clear")}</span>
-            </ButtonV2>
-            <ButtonV2 ghost onClick={onApply} id="apply-filter">
+            </Button>
+            <Button variant="alert" onClick={onApply} id="apply-filter">
```

Comment:
This was primary, why was it changed to alert?

```suggestion
            <Button variant="primary_outline" onClick={onApply} id="apply-filter">
```